src/Error/Error.js

```diff
@@ -1,9 +1,24 @@
 import React from 'react';
 import './Error.css';
+import brokenVideo from '../images/broken-video.svg';
+import oops from '../images/oops.svg';
 
-const Error = () => {
+const Error = ({ error, errorTrailer }) => {
   return (
-    <h1 className='error'>Opps! Something went wrong, please refresh and try again.</h1>
+    <>
+      { error &&
+        <>
+          <img className='error-icon' src={oops} alt='unhappy oops face icon'></img>
+          <h1 className='error-message'>Something went wrong, please refresh and try again.</h1>
+        </>
+      }
+      {errorTrailer &&
```

Comment:
I like the addition of a second error separately handling the trailer and not breaking the entire page.